File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragments/NoteDetailsFragment.java
package com.github.android.lvrn.lvrnproject.view.fragments;


import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.design.widget.FloatingActionButton;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageButton;

import com.github.android.lvrn.lvrnproject.R;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;


/**
 * @author Andrii Bei <psihey1@gmail.com>
 */

public class NoteDetailsFragment extends Fragment {
    @BindView(R.id.im_btn_arrow_back) ImageButton mImageButtonBack;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_note_details,container,false);
        ButterKnife.bind(this,rootView);
        reInitBaseView();
        return rootView;
    }

    private void reInitBaseView() {
        ((AppCompatActivity)getActivity()).getSupportActionBar().hide();
        FloatingActionButton fa =(FloatingActionButton)(( getActivity()).findViewById(R.id.fab));
        fa.hide();
    }

    @OnClick(R.id.im_btn_arrow_back)
    public void backToPreviousFragment(){
        getActivity().onBackPressed();
    }

}

File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/notedetails/NoteDetailsFragment.kt
package com.github.android.lvrn.lvrnproject.view.fragment.notedetails

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import androidx.fragment.app.activityViewModels
import com.github.android.lvrn.lvrnproject.databinding.FragmentNoteDetailsBinding
import com.github.android.lvrn.lvrnproject.view.activity.notedetail.NoteViewModel
import com.github.android.lvrn.lvrnproject.view.util.convertMillisecondsToString
import com.github.valhallalabs.laverna.persistent.entity.Note

class NoteDetailsFragment : Fragment() {

    private var fragmentNoteDetailsBinding: FragmentNoteDetailsBinding? = null
    private val viewModel: NoteViewModel by activityViewModels()
    private var mSelectNote: Note? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        fragmentNoteDetailsBinding = FragmentNoteDetailsBinding.inflate(inflater, container, false)
        mSelectNote = viewModel.note.value
        getParcelableDataAndSetInView()
        return fragmentNoteDetailsBinding?.getRoot()
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        fragmentNoteDetailsBinding!!.imBtnArrowBack.setOnClickListener { view1 -> backToPreviousFragment() }
    }

    /**
     * A method which hears when user click on button and goes one fragment below from current
     */
    fun backToPreviousFragment() {
        activity?.onBackPressed()
    }

    /**
     * A method which gets data from fragment argument bundles and sets their in defined views
     */
    private fun getParcelableDataAndSetInView() {
        fragmentNoteDetailsBinding!!.tvUpdateDate.text = convertMillisecondsToString(mSelectNote!!.updateTime)
        fragmentNoteDetailsBinding!!.tvCreateDate.text = convertMillisecondsToString(mSelectNote!!.creationTime)
        fragmentNoteDetailsBinding!!.tvNotebookNameDetailNote.text = viewModel.notebook.value?.name
    }
}